mediathread/main/views.py

```diff
@@ -811,7 +814,8 @@ def send_faculty_email(form, faculty_user):
             subject,
             body,
             settings.SERVER_EMAIL,
-            [faculty_user.email])
+            [faculty_user.email],
+            fail_silently=not settings.DEBUG)
```

Comment:
Yeah, I mean, ideally, we'd somehow want to know about the error while still allowing the user to continue with the flow.